packages/core/src/electron-browser/menu/electron-commands.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable } from "inversify"
+import { CommandContribution, CommandRegistry } from "../../common"
+import * as electron from "electron"
+
+@injectable()
+export class ElectronCommandHandlers implements CommandContribution {
+
+    constructor(
```

Comment:
Thanks! Other than this, looks great and works correctly, so please merge once you are done.